scripts/exregional_process_lightning.sh

```diff
@@ -250,6 +250,7 @@ fi
 if [[ "$run_lightning" == true ]]; then
   $APRUN ./${exect} < namelist.lightning > stdout 2>&1 || print_err_msg "\
   Call to executable to run lightning (nc) process returned with nonzero exit code."
+  cp stdout $comout/stdout.t${HH}z.lightning
```

Comment:
Should we also copy the final product into $COMOUT along with stdout?

Reply:
Added